README.md

```diff
@@ -394,10 +396,57 @@ The `extract` method takes no arguments:
     1. `v` must have the same type that `f` returns in `extend`.
 
 
+### Bifunctor
 
+A value that implements the Bifunctor specification is a functor with two
+type arguments that are both covariant.
```

Comment:
It's just my opinion but I think spec should focus on restrictions, explanations of types should live elsewhere. So if it's a restriction it should go into laws in some form. But I think it's more like an explanation.